packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx

```diff
@@ -376,9 +376,12 @@ const Template: React.FC<Props> = (props) => {
                 data-e2e='earnInterestSupplyInformation'
                 fullwidth
                 nature='primary'
-                onClick={() =>
+                onClick={() => {
                   analyticsActions.logEvent(INTEREST_EVENTS.SETTINGS.SUPPLY_INFORMATION)
-                }
+                  /* interestActions.handleWithdrawalSupplyInformation({
+                    origin: 'Settings'
+                  }) */
+                }}
```

Comment:
maybe delete this one since it's not in use?

Reply:
This one is actually just not used because this is going to be on `v2`, that's why it's commented